scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/utils/ForkerUtils.java

```diff
@@ -70,6 +97,17 @@
         }
     }
 
+    private ForkerUtils() {
+
+    }
+
+    public static ForkerUtils getInstance() {
+        if (instance == null) {
+            instance = new ForkerUtils();
+        }
+        return instance;
+    }
+
     public enum ForkMethod {
         NONE("none"),
         PWD("pwd"),
```

Comment:
Why don't we have A ForkMethod called "KEY" in the enumeration?

Reply:
lol, expand the code, there is one.